### PR TITLE
Update 4.3 Testing for configuration management

### PR DESCRIPTION
**Pull request by jzold**
This PR covers issue #62 .

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

Update document 4.3 Testing for configuration management.md with new content 4.3.10 Test for subdomain takeover (OTG-CONFIG-010)

Align formatting of 4.3 Testing for configuration management.md headers

I'll also submit a PR to align links with 3.9 and 3.10 shortly

### CODE REVIEW

**Comment by ThunderSon** [comment]
The links are broken as they are not HTML encoded. @kingthorin we need to decide on how to name the files, and we can do it section by section. What do you think about it, should we start pushing forward?

File: document/4 Web Application Security Testing/4.3 Configuration and Deployment Management Testing/4.3 Testing for configuration management.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Understanding the deployed configuration of the server hosting the web application is almost as important as the application security testing itself. After all, an application chain is only as strong as its weakest link. Application platforms are wide and varied, but some key platform configuration errors can compromise the application in the same way an unsecured application can compromise the server.
 
 In order to evaluate the readiness of the application platform, testing for configuration management includes the following sections:\
-[4.3.1 Test Network/Infrastructure Configuration (OTG-CONFIG-001) ](<./Test_Network/Infrastructure_Configuration_(OTG-CONFIG-001)>)
+[4.3.1 Test Network/Infrastructure Configuration (OTG-CONFIG-001) ](<./4.3.1 Test_Network/Infrastructure_Configuration_(OTG-CONFIG-001)>)
```
Comment (kingthorin) [inline document/4 Web Application Security Testing/4.3 Configuration and Deployment Management Testing/4.3 Testing for configuration management.md]: You could drop all the angle brackets &lt; &gt;

Comment (jzold) [reply document/4 Web Application Security Testing/4.3 Configuration and Deployment Management Testing/4.3 Testing for configuration management.md]: Hi,

will resubmit with a fix for the links shortly

**Comment by jzold** [comment]
should be fine now

**Comment by kingthorin** [comment]
@jzold thanks!

You can either let me know how you want to appear in the credits and I'll add you to: https://github.com/OWASP/OWASP-Testing-Guide-v5/pull/60 or you'll need to remember to add yourself after that's merged

**Comment by jzold** [comment]
@kingthorin thanks, please use the following:
Janos Zold
https://www.linkedin.com/in/janoszold/

**Pull request merged**